kuronometer-core/src/main/scala/com/github/pedrovgs/kuronometer/free/domain/BuildExecution.scala

```diff
@@ -0,0 +1,37 @@
+package com.github.pedrovgs.kuronometer.free.domain
+
+object BuildTool extends Enumeration {
+  val Gradle, Maven, SBT = Value
+}
+
+object Platform extends Enumeration {
```

Comment:
Even when we should not couple the network layer to our domain I use this ``Platform`` class as an enumeration because the [json library used](https://github.com/lift/lift/tree/master/framework/lift-base/lift-json) does not support enumerations implemented with sealed abstract classes.